uamqp/client.py

```diff
@@ -386,7 +386,7 @@ def client_ready(self):
             return False
         return True
 
-    def do_work(self):
+    def do_work(self, **kwargs): # pylint: disable=unused-argument
```

Comment:
This needs to be added to `ClientAsync.do_work_async`

Reply:
updated